07-regexp-mandatory-tasks/src/com/fsher/exercise2/Exercise2.java

```diff
@@ -0,0 +1,13 @@
+package com.fsher.exercise2;
+
+public class Exercise2 {
+    private static final String REGEXP_STRING = "     Removing     Extra Spaces    Between Words in    the text      ";
+
+    public static void main(String[] args) {
+        System.out.println(
+                REGEXP_STRING
+                        .replaceAll("(^[\\s]+)|([\\s]+$)", "")
+                        .replaceAll("(\\s{2,})", " ")
```

Comment:
Combine in a single regex?
`(^\\s+)|(\\s+$)|(\\s{2,})`

Reply:
We need to perform different operations to these regexes